src/Meziantou.Framework.WPF/Collections/BatchMode.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Meziantou.Framework.WPF.Collections
+{
+    public enum BatchMode
+    {
+        Default,
```

Comment:
I think we should rename values
- DeferEvents
- CombineEvents
- ResetCollection

What do you think?